katportalclient/client.py

```diff
@@ -30,12 +28,7 @@
 
 # Limit for sensor history queries, in order to preserve memory on katportal.
 MAX_SAMPLES_PER_HISTORY_QUERY = 1000000
```

Comment:
How does this nr relate to the limit you ahve put on the query. I would think it needs to be the same, or you should fall back to use the limit in katstore query and now re-specify it here. Unless there is a reason for this, like you want to expose the limit to external users of the code?

Reply:
this currently a legacy and left there to be used in the limit parameter in the  http requests. This will either be changed or removed based on what the new strategy towards quertying katstore will be.

Reply:
Ok

Reply:
`MAX_SAMPLES_PER_HISTORY_QUERY` does align with `DEFAULT_LIMIT` in katstore.

But I feel the limit should be removed and a better solution should be implemented on the `katstore_webserver` e.g (`statement_timeout` on the SQL query side) 
The `statement_timeout` will solve the issue where the period you are querying returns more data than the limit

This does introduce a problem when a query for a long period could possibly return a large response and affect the servers memory but that is less of a problem than returning `1000000` where the expected results for the period is `>1000000`

Reply:
Well, then we keep this limit for now.  We can improve at a later stage.  Limiting at 1M samples is how katportalclient behaves currently, so not unexpected for users.